### DISCUSSION TITLE
How to make console errors not break tests with latest version?

### DISCUSSION THREAD

**Discussion by piotrpog** [cypress-io/cypress#23277] [General]
In earlier versions, we could put this in `support/index.js`, but it no longer works:

```
import './commands'
Cypress.on('uncaught:exception', (err, runnable) => {
  // returning false here prevents Cypress from failing the test
  return false
})
```
I also tried to put this file in `e2e` directorty, but this also didnt worked.

**Comment by stavares843** [comment]
hello! 👋 

on Cypress 10 should be `support/e2e.js` instead of `support/index.js`

the same code works here
